backend/src/models/preferencesModel.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose');
+
+const PreferencesSchema = new mongoose.Schema({
+    userID: { 
+        type: mongoose.Schema.Types.ObjectId, 
+        required: true 
+    },
+    minPrice: {
+        type: Number, 
+        default: 0, 
+        required: true
+    },
+    maxPrice: {
+        type: Number, 
+        required: true
+    },
+    housingType: {
+        type: String, 
+        enum: ['studio', '1-bedroom', '2-bedroom', 'other'], 
+        required: true
+    },
+    roommateCount: {
+        type: Number, 
+        required: true
+    },
+    petFriendly: { 
+        type: Boolean, 
+        required: true 
+    },
+    smoker: {
```

Comment:
Would 'smoking' work better for this? (For consistency with partying and drinking)

Reply:
Correct, will address!